### PR TITLE
Update markup in draft-camelot-holy-granade

### PR DESCRIPTION
**Pull request by anermina**
As requested in https://github.com/metanorma/rfc-camelot-holy-grenade/issues/16.

### CODE REVIEW

File: sources/draft-camelot-holy-grenade.adoc [hunk 1]
```diff
@@ -358,58 +234,22 @@ recount here through monospace font and multiple spaces.
...
-[.comment]
-tag::relref1[]
-// tag::relref[]
-////
-
 The *Killer Rabbit of Caerbannog*, that most formidable foe of
 the Knights and of all that is holy or carrot-like, has been
 depicted diversely in lay and in song. We venture to say,
-_contra_ the claim made in <<RFC8140,of,section=4.1>>,
+_contra_ the claim made in <<RFC8140,section=4.1>> of Ze Vompyre,
```
Comment (opoudjis) [inline sources/draft-camelot-holy-grenade.adoc:240]: The "of," markup I have used is legal IETF-specific metanorma: https://www.metanorma.org/author/ietf/topics/markup/#citations

File: sources/draft-camelot-holy-grenade.adoc [hunk 2]
```diff
@@ -561,24 +372,10 @@ tag::figure2[]
...
 .The Sovereign's Orb made invisible
 [link=https://upload.wikimedia.org/wikipedia/commons/b/bf/Coa_Illustration_Elements_Globus_cruciger.svg,align=right]
-image::cruciger.svg[Orb,124,135]
-
-////
-[.comment]
-tag::index1[]
-// tag::index[]
-////
+image::cruciger.svg[]
```
Comment (opoudjis) [inline sources/draft-camelot-holy-grenade.adoc:378]: The attributes on image::[] are legal (alt, height, width); do they cause a problem?

**Review by opoudjis — APPROVED** [review]
As ever, very impressed!

The comments were there to enable extraction of Asciidoctor snippets for a larger document embedding this as a citation, but that is obsolete, and the comments would just get in the way now.

**Comment by ronaldtse** [comment]
There's a crash due to relaton-render:

```
undefined method `children' for #<RelatonBib::FormattedRef:0x000000012632e9d0 @format="application/x-isodoc+xml", @language=nil, @script=nil, @content="G. Chapman, J. Cleese, E. Idle, T. Gilliam, T. Jones, M. Palin. 1975. <em>Monty Python and the Holy Grail</em>.">

          return embedded ? f.children.to_xml : doc.to_xml
                             ^^^^^^^^^
/Users/me/.asdf/installs/ruby/3.1.2/lib/ruby/gems/3.1.0/gems/relaton-render-0.6.1/lib/relaton/render/general/render.rb:119:in `parse'
/Users/me/.asdf/installs/ruby/3.1.2/lib/ruby/gems/3.1.0/gems/relaton-render-0.6.1/lib/relaton/render/general/render.rb:114:in `render'
/Users/me/.asdf/installs/ruby/3.1.2/lib/ruby/gems/3.1.0/gems/metanorma-ietf-3.2.4/lib/isodoc/ietf/references.rb:68:in `bibitem_render'
/Users/me/.asdf/installs/ruby/3.1.2/lib/ruby/gems/3.1.0/gems/metanorma-ietf-3.2.4/lib/isodoc/ietf/references.rb:60:in `block in ietf_bibitem'
/Users/me/.asdf/installs/ruby/3.1.2/lib/ruby/gems/3.1.0/gems/nokogiri-1.14.5-arm64-darwin/lib/nokogiri/xml/builder.rb:433:in `insert'
/Users/me/.asdf/installs/ruby/3.1.2/lib/ruby/gems/3.1.0/gems/nokogiri-1.14.5-arm64-darwin/lib/nokogiri/xml/builder.rb:415:in `method_missing'
```

File: sources/draft-camelot-holy-grenade.adoc (hunk 1 above)
Comment (anermina) [reply sources/draft-camelot-holy-grenade.adoc:240]: Thank you, @opoudjis ! I did not use it before.

File: sources/draft-camelot-holy-grenade.adoc (hunk 2 above)
Comment (anermina) [reply sources/draft-camelot-holy-grenade.adoc:378]: It seems this does not cause a problem, at least I am not reproducing it now.

**Comment by opoudjis** [comment]
Have just fixed crash.

**Pull request merged**